### PR TITLE
Adds user visit count

### PR DESCRIPTION
**Pull request by Willianvdv**

### CODE REVIEW

File: bin/rake [hunk 1]
```diff
@@ -1,8 +1,4 @@
 #!/usr/bin/env ruby
-begin
```
Comment (jobertabma) [inline bin/rake:2]: Why did you remove this in this PR?

File: bin/rails [hunk 2]
```diff
@@ -1,8 +1,4 @@
 #!/usr/bin/env ruby
-begin
```
Comment (jobertabma) [inline bin/rails:2]: Same.

Comment (Willianvdv) [inline bin/rails:2]: Not sure, will remove it from this PR
